packages/devextreme/testing/tests/DevExpress.ui.widgets/button.tests.js

```diff
@@ -813,4 +815,99 @@ QUnit.module('Button', function() {
             });
         });
     });
+
+
+    QUnit.module('Renovaion', {
+        Button(options = {}) {
+            $('#button').dxButton(options);
+            return $('#button');
+        }
```

Comment:
[refactoring]
This approach is not usual for us and only raise questions - it was strange to me to find `this.Button(` call in the tests. It's better to use a default approach for component initialization to keep tests consistent and readable (for old devs at least :) )

Also test naming is pure.

However, I see that u've just copied tests from `Renovation/button.tests`, so feel free to leave this refactoring to us.

Reply:
> However, I see that u've just copied tests from Renovation/button.tests, so feel free to leave this refactoring to us.

Yep, leaving it to you. I just moved tests as we want to delete all Renovation/* files